lib/TickLib.sol

```diff
@@ -493,7 +483,6 @@ library FieldLib {
   // Will throw if field is empty
   function lastOnePosition(Field field) internal pure returns (uint) {
     unchecked {
-      // FIXME stop checking for 0 or integrate it into ctz function in assembly
       require(!field.isEmpty(), "field is 0");
```

Comment:
same here

Reply:
fixed in 962779950c4407e50d1a2c9882e0f1ff6bfb4d56

however all code that calls `[first|last]OnePosition` should be wary about this